<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="recvEdit">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="seialCb"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="baundrateCb">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="dateCb">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QComboBox" name="stopCb">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QComboBox" name="checkCb">
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>30</y>
            <width>431</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>自制串口助手</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sendEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="openBt">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeBt">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendBt">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="clearBt">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
